Dockerfile

```diff
@@ -82,4 +82,6 @@ RUN yarn install --production=false
 
 COPY . .
 RUN chown -R jenkins:jenkins /application
+RUN chmod -R 777 /application
```

Comment:
So on the cluster I am building on the Jenkins user is a different UID, I initially had changed the docker file to use the UID on the different Jenkins cluster, but I didn't want to change the existing build path.

As to why this occurs, we don't have a properly implemented "out of source" build with build artifacts getting written to both /application and $HOME. We could retool everything to only write artifacts to a specific directory and then open up that directory, but I don't know enough about the NPM build stack to comment on how hard that would be. I get around $HOME by setting it to be something world writable in Jenkinsfile.pr.